src/dev/src/app/tree-view/eager-declarative-tree/eager-declarative-tree.ts

```diff
@@ -13,4 +13,25 @@ import { Component } from '@angular/core';
 export class EagerDeclarativeTreeDemo {
   expanded1 = true;
   expanded2 = true;
+  isNodeAdisabled: boolean = true;
+
+  isNodeA1disabled: boolean = false;
+  isNodeA2disabled: boolean = false;
+  isNodeA3disabled: boolean = false;
+
+  isNodeA23disabled: boolean = false;
+
+  toggleNode(name) {
+    if (name === 'A-1') {
+      this.isNodeA1disabled = !this.isNodeA1disabled;
+    } else if (name === 'A-2') {
```

Comment:
This demo is a bit too messy for my taste. We try to have our demos somewhat similar to what we expect our consumers to be doing in real-life application, and a long series of chained `else if` with hardcoded values, and corresponding manual properties with hardcoded names is just not realistic for actual applications.

I know it doesn't change much, but just making this demo a tiny bit more dynamic would go a long way for future maintenance and debugging. Thank you. 🙂